apps/native-component-list/src/screens/AV/AudioPlayer.tsx

```diff
@@ -74,7 +76,10 @@ export default class AudioPlayer extends React.Component<Props, State> {
   _loadSoundAsync = async (source: PlaybackSource) => {
     const soundObject = new Audio.Sound();
     try {
-      await soundObject.loadAsync(source, { progressUpdateIntervalMillis: 150 });
+      await soundObject.loadAsync(source, {
+        progressUpdateIntervalMillis: 150,
+        androidImplementation: 'MediaPlayer',
```

Comment:
Should i attempt to make this toggleable instead of just removing it?

Reply:
Yeah, let's add the ability to switch the implementation then 👍